testing/rabbitmq/rabbitmq.initd

```diff
@@ -0,0 +1,38 @@
+#!/sbin/openrc-run
+
+pidfile=/var/run/rabbitmq/rabbitmq.pid
```

Comment:
If this is a rule, there are a lot of APKBUILD files that violate this rule.  It is a trivial change, but it's also a trivial complaint.